controllers/servicedefaults_controller_test.go

```diff
@@ -297,7 +297,8 @@ func TestServiceDefaultsController_errorUpdatesSyncStatus(t *testing.T) {
 			Name:      svcDefaults.Name,
 		},
 	})
-	req.EqualError(err, "Get \"http://incorrect-address/v1/config/service-defaults/foo\": dial tcp: lookup incorrect-address on 127.0.0.11:53: no such host")
+	req.Error(err)
+	req.Contains(err.Error(), "Get \"http://incorrect-address/v1/config/service-defaults/foo\": dial tcp: lookup incorrect-address")
```

Comment:
I'm running go 1.15 now and these errors are a bit different. If we remove this part then they should work with 1.14 and 1.15